src/actions/level/Level.js

```diff
@@ -0,0 +1,27 @@
+import math from "mathjs";
+import {OperationFactory} from "../operations/OperationFactory";
+
+
+export class Level {
+    constructor(number, categories, categoriesProbability) {
+        this._number = number;
+        this._categories = categories;
+        this._categoriesProbability = categoriesProbability;
+    }
+
+    allPossibleOperations() {
+        let operations = [];
+        for (let category of this._categories){
```

Comment:
@MaxiSuppes Acá de nuevo, el tema del for

Reply:
@lucianolev Cambiado